pipelines/performance-tests-pipeline.yml

```diff
@@ -39,10 +39,9 @@ resources:
 - name: census-rm-terraform
   type: git
   source:
-    branch: ((terraform-branch))
+    branch: perf-resource-workaround
```

Comment:
Could pass this branch in as config but I don't mind just editing the pipeline as it's a temporary hack.

Reply:
yeah hopefully wont need this for long 🤞